### ISSUE TITLE
The pack and import command is failing when i add power FX buttons in Model driven

### ISSUE THREAD

**Issue by sekadiv** [scottdurow/SparkleXrm#444]
The pack and import command is failing when i add power FX buttons in Model driven.  Please try i searched log but this issue is not logging in log but nothing is happening if i have power fx buttons in the App.

**Comment by scottdurow** [comment]
which version of spkl are you using? Can you check that you have version >=1.0.640

When you unpack a solution with Power Fx buttons - you should see a folder called CanvasApps that contains the command library. You can unpack that msapp using the pac canvas unpack command if you want.

**Comment by sekadiv** [comment]
Hi Scottdurow/Sparklexrm

Yes when I unpack it has canvas app folder but when I pack then it’s not
doing it and not much in error log .

**Comment by scottdurow** [comment]
Which version of Microsoft.CrmSdk.CoreTools do you have installed?

**Comment by sekadiv** [comment]
yes i have latest one 9.1.0.90 this is what i have  but still pack is not working with power fx buttons solutions

**Comment by sekadiv** [comment]
unpacking is working fine but when i try to pack it then the issue is coming

**Comment by scottdurow** [comment]
Which step do you think it is failing at - pack or import - can you change the spkl command to just pack?
`"%spkl_path%" pack "%cd%\.." "" %*`

Could you post what you see at the command line?

**Comment by sekadiv** [comment]
Yes I changed to pack in the batch file it’s failing while packing in the
command line but no details of error in the log .

**Comment by scottdurow** [comment]
I have solutions packing and importing Commanding V2 buttons - it sounds like there is something in your solution that is causing SolutionPackager to error out.

Can you try using SolutionPackager directly and see if you get any more info that way - https://docs.microsoft.com/en-us/power-platform/alm/solution-packager-tool

**Comment by sekadiv** [comment]
I have more than one Command buttons and it’s creating .msapp file for each
I think that could be issue while packing .

**Comment by scottdurow** [comment]
That doesn't seem right. I get a single component library per model-driven app with multiple components inside.
